docs/sources/outgoing-webhooks/_index.md

```diff
@@ -13,6 +13,15 @@ weight: 500
 
 # Outgoing Webhooks
 
+> ⚠️ A note about **(Legacy)** webhooks:  Webhooks that were created before version **v1.3.11** are marked as
+> **(Legacy)**.  Do not worry! It is still connected to any escalation chains and will continue to execute as it
```

Comment:
"They are still connected to their respective escalation chains and will continue to to execute as they always have"?